### DISCUSSION TITLE
編集リクエストを送信しようとしたら「に投稿禁止の内容が含まれています。」というエラーが表示される

### DISCUSSION THREAD

**Discussion by shiracamus** [increments/qiita-discussions#504] [要望・アイデア・不具合]
本文中に修正すべき部分があったので編集リクエストを送信しようとしたら「に投稿禁止の内容が含まれています。」というエラーが出ました。
どの部分がいけないのかと思い、記事内容をすべて消去して「テスト」だけの本文にしても同じエラーがでますし、本文を変更せずにタイトルだけを変更しようとしても同じエラーが出ました。
不具合が発生しているのでしょうか？

### 編集リクエストしようとした記事

https://qiita.com/gen4pen/items/653e1ffd4b5d7f2dee72

### エラーメッセージ

試しにタイトルだけ変更する編集リクエストを送信しようとしてみると、画面下部赤い部分にエラーメッセージが表示されました。
WebブラウザはChromeとFirefoxの両方で試しましたが同じでした。

![image](https://github.com/increments/qiita-discussions/assets/4151943/c3296efb-09a0-4231-af89-c7e7cd6768f5)

**Comment by tanaka-asami415** [comment]
@shiracamus 
ご不便をおかけして、申し訳ありません。

確認したところ、機械学習によるスパム判定の誤検知により、執筆者が行動禁止状態になってしまっていたようでした。
ご本人から問い合わせがあり現在は解除していますので、編集リクエストは送れるようになっていると思われます。

ただ今回のような場合、編集リクエストを送るときの挙動が分かりづらいと思われるので、改善も検討したいと思います🙇

**Reply by shiracamus** [reply]
編集リクエストを出すことができました。ありがとうございました。
